server/test/controllers/ti/TrustedIntermediaryControllerTest.java

```diff
@@ -36,7 +36,7 @@ public void setup() {
   }
 
   @Test(expected = NullPointerException.class)
```

Comment:
Please leave a TODO against the cleanup issue so that we can track what needs to be done still.

Even if we keep this as expecting a null-pointer exception, I'd move it closer to where it's thrown so we can assert. As it stands, I'm not sure where it'd be thrown and it's unclear as to whether the assertions below are even triggered or not. You can use `assertThrows`, which we use throughout unit tests.

Reply:
Unresolving since this doesn't appear to have been addressed. Mind adding a TODO and consider using `assertThrows` instead of catching it at the top-level?